layouts/partials/dynamic_instrumentation/dynamic-instrumentation-languages.html

```diff
@@ -30,6 +30,20 @@
           </div>
         </a>
       </div>
+      <div class="col">
+        <a class="card h-100" href="/dynamic_instrumentation/enabling/ruby">
+          <div class="card-body text-center py-2 px-1">
+            {{ partial "img.html" (dict "root" . "src" "integrations_logos/ruby-preview.png" "class" "img-fluid" "alt" "Ruby" "width" "400") }}
+          </div>
+        </a>
+      </div>
+      <div class="col">
+        <a class="card h-100" href="/dynamic_instrumentation/enabling/nodejs">
+          <div class="card-body text-center py-2 px-1">
+            {{ partial "img.html" (dict "root" . "src" "integrations_logos/nodejs-preview.png" "class" "img-fluid" "alt" "Node.js" "width" "400") }}
```

Comment:
@sstonehill12 clicking the `nodejs` takes me to a page/tab that doesnt exist: https://docs-staging.datadoghq.com/sarah.stonehill/di-main-page-doc-nodejs-ruby/dynamic_instrumentation/enabling/nodejs

Reply:
@sstonehill12 we should merge #27150 before this one

Reply:
It's now merged